passes/techmap/extract_reduce.cc

```diff
@@ -143,6 +164,11 @@ struct ExtractReducePass : public Pass
 				}
 
 				log("  Head cell is %s\n", head_cell->name.c_str());
+				int ha_width = head_cell->getParam("\\A_WIDTH").as_int();
```

Comment:
This seems (extra) redundant. The previous loop to find the sink node checks for the widths, and the subsequent merging operation also checks the widths. Is this actually necessary?